catalog/designers/udifoundry/info.pb

```diff
@@ -0,0 +1,5 @@
+designer: "Ụdị Foundry"
```

Comment:
Perhaps we should wait for an Garret's answer on #7151 to decide for these special characters?

Reply:
This dotbelow seems to be well supported, but yes, I'll check with him!